index.bs

```diff
@@ -10754,11 +10754,26 @@ operation such as resolution of {{Promise}}s in the {{AudioWorkletGlobalScope}}.
 				1. If <var>node</var> is a {{DelayNode}} that is part of a cycle, add it
 					to <var>cycle breakers</var> and remove it from <var>nodes</var>.
 
-			5. If <var>nodes</var> contains cycles, [=mute=] all the
+			5. For each {{DelayNode}} <var>delay</var> in <var>cycle breakers</var>:
+
+				1. Let <var>delayWriter</var> and <var>delayReader</var> be two
+					placeholder {{AudioNode}} that represent the two ends of the delay
```

Comment:
I don't think we discussed this anywhere, this is just prose that describes what the correct behaviour should be.

It's conceptually exactly the same as `tapin~` and `tapout~` though (and iirc that's Max/MSP, altough it's been quite a few years for me 😄). It's a simple way to describe the correct ordering. The alternative is to have `delay_read` and `delay_write` methods to call in the algorithm below, which is complicated enough as it is. 

This way of doing thigns allows separating concerns: sorting out the cycles first (both those that work because they include a delay, and those to mute because they don't), and take care of the rendering later.